File: SoloLearn Codes/Duty Free/DutyFree.cs
using System;
using System.Linq;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(Array
                .ConvertAll(Console.ReadLine()
                .Split(), double.Parse)
                .Any(x => x * 1.1 > 20) ? "Back to the store" : "On to the terminal");
        }
    }
}

File: SoloLearn Codes/Duty Free/DutyFree.java
import java.util.Arrays;
import java.util.Scanner;

public class DutyFree
{
    public static void main(String[] args) {
        System.out.println(Arrays.stream(new Scanner(System.in).nextLine().split(" "))
                .anyMatch(a -> Double.parseDouble(a) >= 20/1.1) ? "Back to the store" : "On to the terminal");
    }
}